### PR TITLE
332/instantiating cow sdk

### PR DESCRIPTION
**Pull request by alfetopito**
# Summary

Part of #332 

Adding SDK instances

  # To Test

Nothing to test

### CODE REVIEW

File: src/custom/constants/index.ts [hunk 1]
```diff
@@ -3,10 +3,13 @@ import { Token, Fraction, Percent } from '@uniswap/sdk-core'
 import { GPv2Settlement, GPv2VaultRelayer } from '@cowprotocol/contracts/networks.json'
 import { WalletInfo, SUPPORTED_WALLETS as SUPPORTED_WALLETS_UNISWAP } from 'constants/wallet'
 
-import { SupportedChainId as ChainId } from 'constants/chains'
+import { SupportedChainId, SupportedChainId as ChainId } from 'constants/chains'
```
Comment (W3stside) [inline src/custom/constants/index.ts:6]: ? why both imports

Comment (alfetopito) [reply src/custom/constants/index.ts:6]: lol, auto-import from IDE I guess

**Review by anxolin — COMMENTED** [review]
Approved

File: src/custom/constants/index.ts [hunk 2]
```diff
@@ -7,6 +7,9 @@ import { SupportedChainId as ChainId } from 'constants/chains'
 import { getAppDataHash } from './appDataHash'
 import ms from 'ms.macro'
 
+import { CowSdk } from '@cowprotocol/cow-sdk'
+import { PINATA_API_KEY, PINATA_SECRET_API_KEY } from 'constants/ipfs'
```
[the remark this answers is not in the document]
Comment (alfetopito) [reply src/custom/constants/index.ts:11]: Yes, we do have them.
They are just being imported here

**Pull request merged**